tidy3d/components/medium.py

```diff
@@ -1450,7 +1455,7 @@ def _eps_inf_greater_no_less_than_one(cls, val):
         if not CustomIsotropicMedium._validate_isreal_dataarray(val):
             raise SetupError("'permittivity' must be real.")
 
-        if np.any(_get_numpy_array(val) < 1):
+        if np.any(_get_numpy_array(val) < 1 - fp_eps):
```

Comment:
Note that in our previous validation `if np.any(_get_numpy_array(val) < 1):`, it can error in a uniform custom medium of eps_inf =1 from unstructured dataset.

Reply:
I don't think we need this change. I think the error comes from how the test is set up, not due to data being an unstructured dataset. See https://github.com/flexcompute/tidy3d/pull/1592